www/admin/inner.js

```diff
@@ -1476,10 +1481,18 @@ Example
 
         var input = h('input', {
             type: 'email',
-            value: ApiConfig.adminEmail || ''
+            value: ApiConfig.adminEmail || '',
+            id: 'emailInput'
         });
         var $input = $(input);
-        var innerDiv = h('div.cp-admin-setter.cp-admin-setlimit-form', input);
+
+        Messages.email = "Email:"; // XXX temporary solution
```

Comment:
it seems like we need a more general solution to labeling all text inputs in the admin general tab